guide/resources/wallet-interoperability.md

```diff
@@ -0,0 +1,196 @@
+
+</div>
+
+## Payment links
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/payment-links.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/payment-links@2x.jpg"
+   alt-text = "Click a link to launch Bitcoin software"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Avoid copying & pasting payment data by providing clickable links with the right information embedded. Bitcoin applications can respond to such link clicks by automatically navigating to the payment screen and prefilling the provided form fields. Payment links as defined in [BIP 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) can contain contain address, amount, label and other details.
+
+Example link:
+```
+bitcoin:175tWpb8K1S7NmH4Zx6rewF9WQrcZv245W?amount=50&label=Luke-Jr&message=Donation%20for%20project%20xyz
+```
+
+</div>
+
+## QR codes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/qr-code.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/qr-code@2x.jpg"
+   alt-text = "A smartphone camera scanning a QR code"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+QR codes (Quick Response codes) are visual representations of data. Since most devices today feature cameras with built-in support for reading QR codes, this technique has become a convenient method to transfer data from one device to another, even if those devices are offline. Common use cases include reading payment invoices (such as the payment links described above), or importing wallet keys from a backup.
+
+Static QR codes can only contain small amounts of information. Animated QR codes address this problem by splitting up the data over multiple static QR codes. Reading animated QR codes is not a standard feature yet.
+
+</div>
+
+## User data import and export
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/user-data-import-export.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/user-data-import-export@2x.jpg"
+   alt-text = "Transfer of user data between applications"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Transaction data is stored on the Bitcoin block chain and available in any wallet a user has set up. As is, this transaction data does not contain any useful information about the reasons why transactions were made, who owns each address, etc. To better understand and organize their finances, users typically enrich transaction data by assigning contacts, notes, labels, and other useful information. This data should be stored in standardized, open formats and easily synced between applications. This is especially useful for users who rely on multiple devices.
```

Comment:
Are there any standards for this we can reference to?

Reply:
Just researched a little and come across some standards that are overly complicated like [ISO 20022](https://en.wikipedia.org/wiki/ISO_20022), [OFX](https://en.wikipedia.org/wiki/Open_Financial_Exchange) and [QIF](https://en.wikipedia.org/wiki/Quicken_Interchange_Format). They are meant for the exchange of financial data, but more on an enterprise/bank level. I wish there was something more lean and JSON-based available.

Reply:
Thanks for looking into it.. For example, I know that Electrum has labels, but it's unclear to me how they can work and can be exported. One day it'll be essential to have a standard in Bitcoin for this sort of data, as ideally nobody wants to loose precious data when switching wallets.